tests/core/test_backend_cache.py

```
@@ -158,17 +146,16 @@ def test_eager_release_mode_closes_backend(self) -> None:
                     mock_backend_class.return_value = mock_backend
 
                     # Acquire and release
-                    pipeline, key = acquire_eager(cfg)
-                    release_eager(key)
+                    pipeline, key = acquire_pipeline(cfg)
```

Ruff (0.15.20)

[warning] Unpacked variable `pipeline` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)